python-requirements.txt

```diff
@@ -29,6 +29,7 @@ pyserial
 pygments
 pytest
 pytest-timeout
+pyusb >= 1.1.0
```

Comment:
Is this a dependency used by the cw310 library? In that case, the dependency should go into the chipwhisperer/cw310 library (here probably: https://github.com/newaetech/chipwhisperer/blob/develop/setup.py#L17)

Reply:
Yes, and thanks for pointing out the repo this should go in. We can remove that commit from this series, then.

Reply:
It looks like the latest library has moved on from pyusb to libusb1. At least chipwhisperer will spit out an informative warning about what's missing, so I'd guess we're okay with skipping this. This commit is removed, so the series is now just the little flag to configure the PLL.